lib/plugins/ramdisk.js

```diff
@@ -75,7 +76,10 @@ module.exports = {
 
     this.log.info(`Ramdisk enabled`);
 
-    const plugin = new WebpackPluginRamdisk({ name: 'wps' });
+    const defaultOptions = { name: 'wps' };
+    const plugin = new WebpackPluginRamdisk(
+      isObject(options) ? { ...options, ...defaultOptions } : defaultOptions
```

Comment:
Maybe better `typeof options === 'object'` without extra package? Arrays are not supported here anyway. Null will be spread without issues.

Reply:
Sure, done.